### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Plebysnacc** [FachschaftMathPhysInfo/pepp#17]
There is some issues to be fixed on our pages:

- checkboxes can not be triggered with ENTER
- Important Elements miss screenreader support

**Comment by Plebysnacc** [comment]
Checkboxes should be reachable by TAB, works in firefox at least. AFAIK aria-labels should be added to important objects etc. 

Ill just push this back until everything else regarding the registrations is done and then have a look on it again

**Comment by Plebysnacc** [comment]
Login Windows is not usable with tabstop and ENTER

**Comment by Plebysnacc** [comment]
Imma rewrite the issue so its easier to follow
